src/pages/Dashboard/Families/FamilyDetail/FamilyInfo.tsx

```diff
@@ -0,0 +1,36 @@
+import type { MantineColor } from '@mantine/core';
+import { Badge, Box, Space, Text, Title } from '@mantine/core';
+
+interface Props {
+  title: string;
+  value: string;
+  isBadge?: boolean;
+  badgeStatus?: Extract<MantineColor, 'green' | 'red' | 'yellow'>;
```

Comment:
and for `badgeStatus` we can use more domain-specific words like `warning`, `error`and `success`

Reply:
@mojtabaesl  Specified them in design doc.